src/librbd/cache/pwl/ssd_types.h

```diff
@@ -0,0 +1,51 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
Nit: rename to `SsdTypes.h` or `SSDTypes.h` to remain consistent w/ remainder of librbd names.